Assets/HoloToolkit/Build/Editor/BuildDeployPortal.cs

```diff
@@ -170,7 +170,7 @@ public static AppDetails QueryAppDetails(string baseAppName, ConnectInfo connect
                 AppList appList = JsonUtility.FromJson<AppList>(appListJSON);
                 for (int i = 0; i < appList.InstalledPackages.Length; ++i)
                 {
-                    string appName = appList.InstalledPackages[i].PackageFullName;
+                    string appName = appList.InstalledPackages[i].Name;
```

Comment:
So, after looking into this a bit more, I don't believe matching against the PackageFullName is the correct approach. It looks like the full package name could be derived from the locally created APPX package, but requires building it yourself - including doing your own hash with the publisher string into an Id (pretty fragile stuff). I think matching against the application name (as the proposed change does) is the better way to go.